tools/maf_cleanup.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/python
+import optparse
+
+### start of parsing parameters
+parser = optparse.OptionParser()
+parser.add_option('-i', '--input-file', action = 'store', dest = 'inputfile', help = "file name of input maf")
+parser.add_option('-o', '--output-file', action= 'store', dest = "outputfile", default=False, help = "name of output maf")
+
+(options, args) = parser.parse_args()
+input_file = options.inputfile
+output_file = options.outputfile
+
+if not input_file:
+	print 'ERROR: missing input MAF name.'
+	parser.print_help()
+elif not output_file:
+	print 'ERROR: missing output MAF name.'
+	parser.print_help()
+
+### end of parsing parameters
+
+### start of searching for header row
+header_line = ""
+with open(input_file, "r") as f:
+	for line in f:
+		values = line.split("\t")
+		if values[0].upper() == "HUGO_SYMBOL":
+			header_line = line
+headers = header_line.rstrip().split("\t")
+keep_col_index_arr = [] #Index of columns to keep
+### end of searching for header row
+
+### start of filtering out unwanted columns
+count = 0
+exon_number_index = -1
+exon_index = -1
+intron_index = -1
+oncotator_index_arr = []
+for header_item in headers:
+	if header_item.upper() == "EXON_NUMBER":
+		exon_number_index = count
+	elif header_item.upper() == "EXON":
+		exon_index = count
+	elif header_item.upper() == "INTRON":
+		intron_index = count
+	elif header_item[0:len("ONCOTATOR")].upper() == "ONCOTATOR":
+		oncotator_index_arr.append(count)
+	else:
+		keep_col_index_arr.append(count)
+	count += 1
+
+print "INDEX OF COLUMNS TO DROP >>>>>>>>>>>>>>"
+print "INTRON: " + str(intron_index)
+print "EXON_NUMBER: " + str(exon_number_index)
+print "EXON: " + str(exon_index)
+print "ONCOTATOR: " + ' '.join(str(index) for index in oncotator_index_arr)
+print "Reconstructing file ...."
+### end of filtering out unwated columns
+
+### Start of reconstructing file
+input_maf = open (input_file, "r")
```

Comment:
remove whitespace between `open` and parentheses

Reply:
fixed